hardware_interface/CMakeLists.txt

```
@@ -70,16 +72,16 @@ if(BUILD_TESTING)
   ament_target_dependencies(test_inst_hardwares rcpputils)
 
   ament_add_gmock(test_joint_handle test/test_handle.cpp)
-  target_link_libraries(test_joint_handle hardware_interface)
-  ament_target_dependencies(test_joint_handle rcpputils)
+  target_link_libraries(test_joint_handle hardware_interface fmt::fmt)
+  ament_target_dependencies(test_joint_handle rcpputils fmt)
 
   ament_add_gmock(test_component_interfaces test/test_component_interfaces.cpp)
-  target_link_libraries(test_component_interfaces hardware_interface)
-  ament_target_dependencies(test_component_interfaces ros2_control_test_assets)
+  target_link_libraries(test_component_interfaces hardware_interface fmt::fmt)
+  ament_target_dependencies(test_component_interfaces ros2_control_test_assets fmt)
 
   ament_add_gmock(test_component_interfaces_custom_export test/test_component_interfaces_custom_export.cpp)
-  target_link_libraries(test_component_interfaces_custom_export hardware_interface)
-  ament_target_dependencies(test_component_interfaces_custom_export ros2_control_test_assets)
+  target_link_libraries(test_component_interfaces_custom_export hardware_interface fmt::fmt)
+  ament_target_dependencies(test_component_interfaces_custom_export ros2_control_test_assets fmt)
```

```suggestion
  target_link_libraries(test_component_interfaces_custom_export hardware_interface)
  ament_target_dependencies(test_component_interfaces_custom_export ros2_control_test_assets)
```
I don't think it is needed in these files

---

fixed